src/reprs/binary32.rkt

```diff
@@ -70,3 +67,24 @@
 
 (define-operator-impl (cast binary32->binary64 binary32) binary64
   [fl identity])
+
+(register-accelerator-implementation!
+ 'expm1 'expm1.f32
+ (list (get-representation 'binary32)) (get-representation 'binary32)
+ (get-ffi-obj 'expm1 #f (_fun _float -> _float) (const #f)))
```

Comment:
Ooh I never considered this issue. Ok so I have some functions in `src/reprs/runtime/libm.rkt` for this. I believe what you should do
```
(define-libm hypot-impl (hypotf float float))
(when hypot-impl
  (register-accelerator-implementation! 'hypot
                                        'hypot.f32
                                        (list (get-representation 'binary32)
                                              (get-representation 'binary32))
                                        (get-representation 'binary32)
                                        hypot-impl))
```
where 
```
(define-libm <id> (<name> <type> ... <type>))
```
binds to `<id>` either the libm procedure named `<name>` (check C standard for the correct name) with type signature `<type> ... <type>` or `#f` if this fails.  

Remember that libm may differ between say Mac and Windows!!! So this can fail. Only register the impl if it exists. You can drop the ffi import then.